test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-sync-delete-policy/99-cleanup.yaml

```diff
@@ -0,0 +1,5 @@
+# This clean-up stage is necessary because of https://github.com/kyverno/kyverno/issues/5101
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl delete ur -A --all
```

Comment:
Once the clusterpolicy gets deleted,  respected UR will be deleted after 5 retries by the controller just to make sure policy is no longer exists in cluster and process with UR cleanup.

Reply:
It's not quick enough if there's another test that proceeds immediately after which uses the same resource names.